pkg/types/types.go

```diff
@@ -330,25 +330,26 @@ type NodeIP struct {
 
 // Node describes a k3d node
 type Node struct {
-	Name       string            `yaml:"name" json:"name,omitempty"`
-	Role       Role              `yaml:"role" json:"role,omitempty"`
-	Image      string            `yaml:"image" json:"image,omitempty"`
-	Volumes    []string          `yaml:"volumes" json:"volumes,omitempty"`
-	Env        []string          `yaml:"env" json:"env,omitempty"`
-	Cmd        []string          // filled automatically based on role
-	Args       []string          `yaml:"extraArgs" json:"extraArgs,omitempty"`
-	Ports      nat.PortMap       `yaml:"portMappings" json:"portMappings,omitempty"`
-	Restart    bool              `yaml:"restart" json:"restart,omitempty"`
-	Created    string            `yaml:"created" json:"created,omitempty"`
-	Labels     map[string]string // filled automatically
-	Networks   []string          // filled automatically
-	ExtraHosts []string          // filled automatically
-	ServerOpts ServerOpts        `yaml:"serverOpts" json:"serverOpts,omitempty"`
-	AgentOpts  AgentOpts         `yaml:"agentOpts" json:"agentOpts,omitempty"`
-	GPURequest string            // filled automatically
-	Memory     string            // filled automatically
-	State      NodeState         // filled automatically
-	IP         NodeIP            // filled automatically
+	Name        string            `yaml:"name" json:"name,omitempty"`
+	Role        Role              `yaml:"role" json:"role,omitempty"`
+	Image       string            `yaml:"image" json:"image,omitempty"`
+	Volumes     []string          `yaml:"volumes" json:"volumes,omitempty"`
+	Env         []string          `yaml:"env" json:"env,omitempty"`
+	Cmd         []string          // filled automatically based on role
+	Args        []string          `yaml:"extraArgs" json:"extraArgs,omitempty"`
+	Ports       nat.PortMap       `yaml:"portMappings" json:"portMappings,omitempty"`
+	Restart     bool              `yaml:"restart" json:"restart,omitempty"`
+	Created     string            `yaml:"created" json:"created,omitempty"`
+	Labels      map[string]string // filled automatically
+	ExtraLabels map[string]string `yaml:"extraLabels" json:"extraLabels,omitempty"`
```

Comment:
```suggestion
	ExtraLabels map[string]string `yaml:"extraLabels" json:"extraLabels,omitempty"`
```
```suggestion
	KubeNodeLabels map[string]string `yaml:"kubeNodeLabels" json:"kubeNodeLabels,omitempty"`
```
I guess `extraLabels` just next to `labels` which refer to docker container labels may be a bit confusing :thinking:

Reply:
Should `Labels` instead be renamed to `DockerLabels`? that would make things normalized with the flags `--node-label` & `--docker-label`

Reply:
It's both labels on nodes... Just depends on whether you're referring to the nodes as viewed by k3d or by k3s.
Also, docker is a runtime implementation in this case and so I'd prefer `--runtime-label` instead.

Reply:
Agree with `--runtime-label` to make it generic. But for the other label it would be good if it goes with the same naming as the flag, and since you proposed `--k3s-node-label` should it be named `K3sNodeLabels`?

Reply:
Sounds good to me :+1:

Reply:
done @iwilltry42, we changed the flag and the variable name like you said.

Reply:
:+1: